confectory-core/src/main/java/net/obvj/confectory/source/ClasspathFileSource.java

```diff
@@ -58,9 +61,13 @@ protected T load(URL url, Mapper<T> mapper)
         try (InputStream inputStream = url.openStream())
         {
             LOGGER.info("Loading file {} with mapper: <{}>", super.source, mapper.getClass().getSimpleName());
+            Stopwatch stopwatch = Stopwatch.createStarted();
             T mappedObject = mapper.apply(inputStream);
+            stopwatch.stop();
+            double elapsedTime = stopwatch.elapsedTime(Counter.Type.WALL_CLOCK_TIME, TimeUnit.MILLISECONDS);
 
             LOGGER.info("File {} loaded successfully", super.source);
+            LOGGER.info(String.format("File loaded in %d ms", (int)Math.round(elapsedTime)), super.source);
```

Comment:
I would prefer to report the elapsed time with the decimal places.
A better way to do that could be getting the elapsed time as `Duration`, like this:

`Duration elapsedTime = stopwatch.elapsedTime(Counter.Type.WALL_CLOCK_TIME);`

then simply relying on the toString() method (which also prints an appropriate legend)

`LOGGER.info("File loaded in {}", duration);`